crates/egui-winit/src/lib.rs

```diff
@@ -766,7 +770,7 @@ impl State {
             }
 
             self.egui_input.events.push(egui::Event::Key {
-                key: logical_key,
+                key: active_key,
```

Comment:
We should document that `Event::Key::key` is either the logical key (if known), else the physical key.

Reply:
thanks for highlighting this -- I have added a clarification. if it's not to your liking, feel free to reword